docs/sections/user_guide/cli/mode_rocoto.rst

```diff
@@ -218,20 +213,20 @@ The examples that follow use a UW YAML file ``rocoto.yaml`` with content
 
   Optional arguments:
     -h, --help
-          Show help and exit
+        Show help and exit
     --input-file PATH, -i PATH
-          Path to input file (defaults to stdin)
+        Path to input file (defaults to stdin)
     --quiet, -q
-          Print no logging messages
+        Print no logging messages
     --verbose, -v
-          Print all logging messages
+        Print all logging messages
 
 Examples
-~~~~~~~~
+^^^^^^^^
 
 The examples that follow use a Rocoto XML file ``rocoto.xml`` with the following content:
 
-.. code-block:: XML
+.. code-block:: xml
```

Comment:
Sphinx doesn't seem to care about the capitalization, but it uses [Pygments](https://pygments.org/languages/) for syntax highlighting, and that tool lists the "short name" for XML as `xml`. So, consistency, least-surprise, etc.